### PR TITLE
Remove questions about ways

### PR DESCRIPTION
**Pull request by 7h30n3**
..,because there are too many of them.

### CODE REVIEW

File: assets/datasets/question_catalog.json [hunk 1]
```diff
@@ -2025,138 +1734,5 @@
...
-      },
-      "conditions": [
-         {
-            "osm_tags": {
-               "conveying": ["yes", "forward", "backward", "reversible"],
-               "duration": false
-            },
-            "osm_element": "OpenWay"
-         }
-      ]
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:2160]: We can keep this.

File: assets/datasets/question_catalog.json [hunk 2]
```diff
@@ -1506,142 +1383,6 @@
...
-               "surface": false
-            },
-            "osm_element": [
-               "OpenWay",
-               "ClosedWay",
-               "Relation"
-            ]
-         }
-      ]
-   },
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:1568]: We should keep this. If not all then at least the `"public_transport": "platform",` part.

**Review by Robbendebiene — CHANGES_REQUESTED** [review]

File: assets/datasets/question_catalog.json (hunk 1 above)
Comment (7h30n3) [reply assets/datasets/question_catalog.json]: Re-added.

File: assets/datasets/question_catalog.json (hunk 2 above)
Comment (7h30n3) [reply assets/datasets/question_catalog.json]: Re-added the `platform`-Part.

**Review by Robbendebiene — APPROVED** [review]
LGTM

**Pull request merged**